File: Tabbar/DEMO/DEMO/ViewController.m
//
//  ViewController.m
//  DEMO
//
//  Created by Alex D. on 3/25/16.
//  Copyright © 2016 ifnil. All rights reserved.
//

#import "ViewController.h"
#import "DEMO-swift.h"

@interface ViewController () <CCTabbarDelegate>
@property (weak, nonatomic) IBOutlet CCTabbar *tabbar;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    UIButton *bt1 = [[UIButton alloc] init];
    [bt1 setTitle:@"节点" forState:UIControlStateNormal];
    [bt1 setTitleColor:UIColor.redColor forState:UIControlStateNormal];
    UIButton *bt2 = [[UIButton alloc] init];
    [bt2 setTitle:@"最新" forState:UIControlStateNormal];
    [bt2 setTitleColor:UIColor.blueColor forState:UIControlStateNormal];
    self.tabbar.models = @[bt1, bt2];
    self.tabbar.delegate = self;
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

#pragma mark - CCTabbarDelegate
- (void)tabbar:(CCTabbar *)tabbar didSelectAtIndex:(NSInteger)index {
    NSLog(@"selected at %ld", (long)index);
}

@end

File: 2048/ViewController.swift
//
//  ViewController.swift
//  2048
//
//  Created by 丁松 on 14-6-24.
//  Copyright (c) 2014年 丁松. All rights reserved.
//

//PlayGround 是一个测试“打草稿”的地方

import UIKit

class ViewController: UIViewController {
                            
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    //按键的功能，弹出对话框，在storyboard中右键关联到这个函数
    @IBAction func StartGame(sender:UIButton){
        let alertView = UIAlertView()
        alertView.title = "开始"
        alertView.message = "游戏开始,准备好吗?"
        alertView.addButtonWithTitle("Ready Go!")
        alertView.show()
    }


}

